src/app/_components/sections/serviceSection.js

```
@@ -1,28 +1,34 @@
+              <video
+                className="w-full h-full object-cover"
+                src={item.videoSrc}
+                autoPlay
+                loop
+                muted
+                playsInline
+              />
+            </div>
+            <TypographyP
+              className="whitespace-pre-line"
+              dangerouslySetInnerHTML={{ __html: item.subtitle }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)